velox/connectors/Connector.h

```diff
@@ -224,9 +224,16 @@ class DataSource {
 /// of the caller.
 class ConnectorQueryCtx {
  public:
+  using ConnectorMemoryPoolFactory =
+      std::function<std::shared_ptr<memory::MemoryPool>(
+          const std::string& name,
+          memory::MemoryPool* parent,
+          bool isLeaf)>;
+
   ConnectorQueryCtx(
       memory::MemoryPool* operatorPool,
       memory::MemoryPool* connectorPool,
```

Comment:
I guess we don't need connectorPool and provide connectorMemoryPoolFactory instead?